kano_updater/scenarios.py

```diff
@@ -452,7 +452,21 @@ def beta_4_1_1_to_beta_4_2_0(self, dummy_progress):
         pass
 
     def beta_4_2_0_to_beta_4_2_1(self, dummy_progress):
-        pass
+        #
+        # On Kano OS 4.2.0 Christmas release, a hotfix was provided to a number of users
+        # to recover from a faulty wireless reconnection on boot, by executing the command below.
+        #
+        # "sudo mv /lib/dhcpcd/dhcpcd-hooks/10-wpa_supplicant /root"
+        #
+        # This step reinstalls the dhcpcd5 package so that the file is restored back,
+        # and the supplicant daemon is started correctly back again.
+        #
```

Comment:
```suggestion
        #
        # Note that in 4.2.1 both kano-settings and kano-toolset were updated to use the original
        # wpa_supplicant. Updater's dependency list was *specifically* not updated to reflect this
        # in order to delay a potential issue for users with the workaround in case of a failed update.
        # When the dependency versions will be bumped, keep this case in mind.
```

Reply:
Included a bit more info about the situation.